### DISCUSSION TITLE
TURBULENT KINETIC ENERGY

### DISCUSSION THREAD

**Discussion by FernandoIGuerra** [AMReX-Combustion/PeleC#777] [Q&A]
is it possible to calculate the turbulent velocity o the turbulent kinetic energy usign DNS? Thanks in advance

**Comment by marchdf** [comment]
Depending on the details of what you are trying to do, you can add a derived field. You can see how that is done for kinetic energy: https://github.com/AMReX-Combustion/PeleC/blob/development/Source/Derive.cpp#L117.
